src/discordtipbot/discordbot.cr

```diff
@@ -128,11 +128,9 @@ class DiscordBot
   end
 
   private def dm_deposit(user : UInt64)
-    begin
-      @bot.create_message(@cache.resolve_dm_channel(user), "Your deposit just went through! Remember: Deposit Addresses are *one-time* use only so you'll have to generate a new address for your next deposit!\n*#{TERMS}")
-    rescue
-      @log.info("#{@config.coinname_short}: Failed to contact #{user} with deposit notification")
-    end
+    @bot.create_message(@cache.resolve_dm_channel(user), "Your deposit just went through! Remember: Deposit Addresses are *one-time* use only so you'll have to generate a new address for your next deposit!\n*#{TERMS}")
+  rescue
+    @log.info("#{@config.coinname_short}: Failed to contact #{user} with deposit notification")
```

Comment:
Is `info` appropriate?  To me you'd at least want to use `warn`. If you're writing your logs to a file, it will make them much easier to grep through for these cases if you needed to.

If you need to contact these people you could probably resolve the the user from the cache in the `rescue` block and print `"#{user.username}##{user.discriminator}"` so it is easier for you to do so. Just a thought.